lib/services/smartling.rb

```diff
@@ -15,7 +15,7 @@ class Service::Smartling < Service
 
   def receive_push
     check_config
-    if data["master_only"] == false || payload["ref"] == "refs/heads/master"
+    if data["master_only"] == nil || data["master_only"] == false || payload["ref"] == "refs/heads/master"

```

Comment:
It looks like you might want to use the new `config_boolean_false?("master_only")` option. You can see that code here: https://github.com/github/github-services/blob/master/lib/service.rb#L506